catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/model/Search.java

```diff
@@ -361,6 +367,15 @@ private void addObject(Map<String, Object> obj, String name, Object value) {
             metacard.put(CACHED, ISO_8601_DATE_FORMAT.print(new DateTime()));
         }
 
+        Optional<Serializable> internalLocalResource = Optional.ofNullable(result.getMetacard()
+                .getAttribute(INTERNAL_LOCAL_RESOURCE))
+                .map(Attribute::getValue);
+        if (internalLocalResource.isPresent()) {
```

Comment:
I would suggest using `orElse()` from Optional instead of `isPresent()`.

http://docs.oracle.com/javase/8/docs/api/java/util/Optional.html#orElse-T-
https://dzone.com/articles/java-8-optional-replace-your-get-calls